src/plugin/file/kml/ui/kmlnode.js

```diff
@@ -893,3 +893,10 @@ plugin.file.kml.ui.KMLNode.prototype.getLayerUI = function(item) {
 
   return null;
 };
+
+/**
+ * @return {os.annotation.FeatureAnnotation}
+ */
+plugin.file.kml.ui.KMLNode.prototype.getAnnotation = function() {
```

Comment:
Please remove this function. The annotation should not be exposed outside the KML node. This is explained further in my comment in the issue.